<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSearchConfig</class>
 <widget class="QDialog" name="DataSearchConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataSearchConfig</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0" colspan="3">
    <widget class="QTabWidget" name="tabConfigs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabConnectivity">
      <attribute name="title">
       <string>Connectivity</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="btnGroup_DBMS">
         <property name="title">
          <string>DBMS</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rad_DBMS_MySql">
            <property name="text">
             <string>MySQL</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rad_DBMS_MSAcces">
            <property name="text">
             <string>MS Acces SQL</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Database driver</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_DBdriver"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Database host</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edt_DBhost">
           <property name="text">
            <string>localhost</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edt_DBport">
           <property name="text">
            <string>3306</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Database</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rad_DB_DSNfile">
            <property name="text">
             <string>DSN file</string>
            </property>
            <attribute name="buttonGroup">
             <string>btnGroup_DataBase</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="edt_DSNfile"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btn_BrowseDsnFile">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="btn_CreateDsnFile">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rad_DB">
            <property name="text">
             <string>Database</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string>btnGroup_DataBase</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="edt_DBname"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>User name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edt_Username">
           <property name="text">
            <string>root</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edt_Password">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>394</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="btnTestConnectivity">
         <property name="text">
          <string>Test connectivity</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTable">
      <attribute name="title">
       <string>Table</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Table name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edt_Tablename"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1" rowspan="3">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Available tables</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>71</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="btn_RefeshTableList">
              <property name="text">
               <string>Refresh list</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_UseTable">
              <property name="text">
               <string>Use table</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>70</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QListWidget" name="lst_Tables"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_CreateTable">
         <property name="text">
          <string>Create table using inbuilt script </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>282</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>288</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="btnGroup_Accept">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btn_Apply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnGroup_Accept</sender>
   <signal>rejected()</signal>
   <receiver>DataSearchConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGroup_Accept</sender>
   <signal>accepted()</signal>
   <receiver>DataSearchConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="btnGroup_DataBase"/>
 </buttongroups>
</ui>
